developer/src/kmanalyze/kmanalyze.cpp

```diff
@@ -155,9 +153,7 @@ BOOL LoadKeyboard(LPSTR fileName, LPKEYBOARD *lpKeyboard)
     ckbp->dwFileVersion > VERSION_MAX)
   {
     /* Old or new version -- identify the desired program version */
-    if((ckbp->dwFileVersion >= VERSION_160 && ckbp->dwCheckSum == 0) ||
-      /* #7222: We support a zero checksum in Keyman 16.0 and later */
-      VerifyChecksum(buf, &kbp->dwCheckSum, sz))
+    if(ckbp->dwFileVersion >= VERSION_160)
     {
```

Comment:
This change is incorrect. The child block should always run. You'll need to remove `if() {` and `}` altogether and always run the child block (and you'll need to fixup indents).

Reply:
good catch, thanks.  I also added brackets around the for.